awx/ui_next/src/components/Lookup/CredentialLookup.jsx

```diff
@@ -111,6 +112,39 @@ function CredentialLookup({
     }
   );
 
+  const checkCredentialName = useCallback(
+    async name => {
+      if (name && name !== '') {
```

Comment:
To un-nest this all a bit, you could handle the `else` case first and return early:

```
if (!name) {
  onChange(null);
  return;
}
...
```

Also, the empty string check is redundant, so probably not necessary

Reply:
Good call - I like that